.github/workflows/simple.yml

```diff
@@ -13,3 +13,7 @@ jobs:
       run: |
         python -c "print(3+2)"
         python --version
+    - name: Get running processes for the current user and get disk usage in human readable format
+      run: 
```

Comment:
Oh, the action fails.
https://github.com/matsui528/actions_test/actions/runs/2158906878

It's because `|` is lacking here. With `|`, you can write [multiple lines as is](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines). Can you add it in another PR?

Reply:
Done!

Reply:
Ah, I need to open another PR.

Reply:
But, the ps -u command doen't work on Windows and build fails on Windows. Anyways, I learnt about PR's and Actions. Cheers! 🎉

Reply:
🎉